usr/share/rear/restore/COVE/default/400_restore_with_cove.sh

```
@@ -308,10 +313,8 @@ cove_print "Waiting for the restore to be started... "
 cove_wait_for 'local status="$(cove_get_status)"; [ "${status}" = "Scanning" -o "${status}" = "Restore" ]' 2 \
     && cove_print_done || { cove_print_error; Error "The restore has not started."; }
 
-# Wait for the restore to be finished
-cove_print "Waiting for the restore to be finished... "
-cove_wait_for 'local status="$(cove_get_status)"; [ "${status}" = "Idle" ]' 15 \
-    && cove_print_done || { cove_print_error; Error "The restore has not finished."; }
+# Show progress bar for restore session
+cove_show_progress
```

Can this command fail, or there is the guarantee that everything have to be OK?

---

Theoretically yes, for example in case BackupFP for some reasons will be crashed or not started.

---

What about cancelling the `ClientTool` via `Ctrl-C`?
I am asking because I guess we need to report an error if the restore session has not finished, I meant `cove_show_progress || Error "..."`

---

Agree, it can be a problem. Added error message